.cirrus/cirrus-sq-analysis.sh

```diff
@@ -40,6 +40,6 @@ if [[ "${PULL_REQUEST}" ]] || [[ "${GITHUB_BRANCH}" == "main" ]]; then
                                 src/components/icons/**,
                                 src/generated/**,
                                 ")
-  sonar-scanner "${scanner_params[@]}"
+  npx sonar-scanner "${scanner_params[@]}"
```

Comment:
does it matter that we run it with npm instead of yarn while all our other commands are run with yarn? I would guess not, just raising the question.

Reply:
because yarn doesn't have a `--no-save` option, I went with npm. But if I install it globally into docker, I'll use yarn and this won't be an issue 👍🏻